netlify.toml

```diff
@@ -0,0 +1,26 @@
+[build]
```

Comment:
It's probably worth a header comment here explaining how we use Netlify (and that it's not used in production)

Reply:
I think once it goes to production there'll be no Netlify anyways?

Reply:
I think we'll still use netlify to test PRs?